tests/models/test_timestamp.py

```diff
@@ -0,0 +1,76 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import pendulum
+import pytest
+from freezegun import freeze_time
+
+from airflow.models import DAG, Log, TaskInstance
+from airflow.operators.dummy_operator import DummyOperator
+from airflow.utils import timezone
+from airflow.utils.session import provide_session
+from airflow.utils.state import State
+
+
+@pytest.yield_fixture()
+@provide_session
+def clear_db(session=None):
```

Comment:
From https://docs.pytest.org/en/latest/reference.html#pytest-fixture
> If a fixture is used in the same module in which it is defined, the function name of the fixture will be shadowed by the function arg that requests the fixture; one way to resolve this is to name the decorated function `fixture_<fixturename>` and then use `@pytest.fixture(name='<fixturename>')`.

```suggestion
@pytest.fixture(name="clear_db")
@provide_session
def fixture_clear_db(session=None):
```

That way we don't need the pylint disables.

Reply:
Nice!

Reply:
![image](https://user-images.githubusercontent.com/595491/76243366-80fd5c80-6238-11ea-9032-fc809cd963ae.png)